### DISCUSSION TITLE
v2.2.0

### DISCUSSION THREAD

**Discussion by donavanbecker** [homebridge-plugins/homebridge-august#112] [Announcements]
#### Major Changes
- Devices that are Homekit enabled will now not display unless the new `overrideHomekitEnabled` device config flag is set.

#### Other Changes
- All for overriding `PushRate` which allows you to set how quick changes are pushed to August API with the `PushRate` config
  - This can be set at device or plugin level
- Added Success Logs(`green`) which requires Homebridge 1.8.0 or higher
- Housekeeping and updated dependencies.

**Full Changelog**: https://github.com/donavanbecker/homebridge-august/compare/v2.1.0...v2.2.0

<hr /><em>This discussion was created from the release <a href='https://github.com/donavanbecker/homebridge-august/releases/tag/v2.2.0'>v2.2.0</a>.</em>

**Comment by taisontan** [comment]
I'm not sure how to complete the new change: `overrideHomekitEnabled` device config flag is set.

Is this something I add to the JSON file?

**Reply by donavanbecker** [reply]
Go to plugin settings. Type in device id and check the flag.

**Reply by donavanbecker** [reply]
![image](https://github.com/donavanbecker/homebridge-august/assets/9875439/37e754a6-acaa-4a06-b94f-42df112b4a0d)

**Reply by taisontan** [reply]
It seems I didn’t configure it properly before the update to 2.2.0. I’ve since done that while at 2.0.0 and then updated to 2.2.0. All is good now. Thank you!
